python/pdo/service_client/storage.py

```diff
@@ -130,7 +130,7 @@ def get_blocks(self, block_ids) :
 
         url = "{0}/block/gets".format(self.ServiceURL)
         try :
-            response = requests.post(url, json=block_ids, timeout=self.default_timeout)
+            response = self.session.post(url, json=block_ids, timeout=self.default_timeout)
```

Comment:
In eservice client, for post requests, I see that you have explicitly set stream=False. And I dont see it here. Not sure what the difference is.